File: 백준/Bronze/10798. 세로읽기/세로읽기.cc
#include <iostream>
#include <string>
using namespace std;

char word[5][16];

int main(){
     for(int i = 0; i < 5; i++){
         cin >> word[i];
     }
     
     for(int i = 0; i < 15; i++){
         for(int j = 0; j < 5; j++){
             if(word[j][i] != '\0') cout << word[j][i];
         }
     }
}

File: 백준/Bronze/10798. 세로읽기/세로읽기.swift
var words = [[String]](repeating: [String](repeating: "", count: 15), count: 5)
var result = ""

for i in 0..<5 {
    let word = readLine()!.map{ String($0) }
    words[i].replaceSubrange(0..<word.count, with: word)
}

for i in 0..<15 {
    for j in 0..<5 {
        print(words[j][i], terminator: "")
    }
}